packages/testlab/src/test-sandbox.ts

```diff
@@ -82,6 +95,12 @@ export class TestSandbox {
     dest = dest
       ? resolve(this.path, dest)
       : resolve(this.path, parse(src).base);
+
     await copy(src, dest);
+
+    if (parse(src).ext === '.js' && pathExists(src + '.map')) {
+      const srcMap = src + '.map';
+      await appendFile(dest, `\n//# sourceMappingURL=${srcMap}`);
```

Comment:
IIUC, this assumes that the `.js` file does not contain any `//# sourceMappingURL` directive yet. Is it a safe assumption? Should we support the case when the `.js` file already contains sourceMappingURL directive?

Reply:
Nope. A file can contain the directive but source-map only considers the last directive for the sourceMappingURL. 

The moment we compile a file it gets this directive ... this adds the directive again with the absolute path of the original map file as the last line (so the previous line is ignored as just a comment) after copying the file over to it's new destination.